pkg/test/resourcefixture/testdata/basic/gkehub/v1beta1/gkehubfeaturemembership/basicacmgkehubfeaturemembership/update.yaml

```diff
@@ -38,6 +38,5 @@ spec:
         syncWaitSecs: "20"
         syncRev: "HEAD"
         secretType: "none"
-    version: "1.18.1"
     management: "MANAGEMENT_AUTOMATIC"
 
```

Comment:
I was chatting with @maqiuyujoyce offline and we were thinking it would be best if we kept this test how it was before -- without adding the `management` field. Then, we could have a separate test called "csmanagement-base` and `csmanagement-full` that showcases the mutual exclusivity of the `management` field and the `version`. Could you please make those changes 🙏🏼 ?

- remove `management` field from these two tests
- regen logs
- add a new test suite that this one without the `version` field but with the `management` field set.